lib/squad_client.py

```diff
@@ -65,24 +65,26 @@ def urljoiner(*args):
     return "/".join(map(lambda x: str(x).rstrip("/"), args))
 
 
-def get_objects(endpoint_url, expect_one=False, parameters={}):
+def get_objects(endpoint_url, parameters={}, limit=0):
```

Comment:
I think I would be happier with this if it defaulted to None

Reply:
Done.